cluster/images/conformance/run_e2e.sh

```diff
@@ -59,7 +59,7 @@ ginkgo_args+=(
 )
 
 set -x
-/usr/local/bin/ginkgo "${ginkgo_args[@]}" /usr/local/bin/e2e.test -- --disable-log-dump --repo-root=/kubernetes --provider="${E2E_PROVIDER}" --report-dir="${RESULTS_DIR}" --kubeconfig="${KUBECONFIG}" | tee "${RESULTS_DIR}"/e2e.log &
+/usr/local/bin/ginkgo "${ginkgo_args[@]}" /usr/local/bin/e2e.test -- --disable-log-dump --repo-root=/kubernetes --provider="${E2E_PROVIDER}" --report-dir="${RESULTS_DIR}" --kubeconfig="${KUBECONFIG}" "${E2E_VERBOSITY}" | tee "${RESULTS_DIR}"/e2e.log &
```

Comment:
`--v="{E2E_VERBOSITY}"` please

we don't want them passing in random flags.

Reply:
Nice point, thanks